docs/validation.md

```diff
@@ -34,17 +34,18 @@ Steps:
     -   `statement` := decode `envelope.payload` as a JSON-encoded
         [Statement]; reject if decoding fails
     -   Reject if `statement.type` != `https://in-toto.io/Statement/v1.0`
-    -   `artifactNames` := empty set of names
+    -   `matchedSubjects` := empty set of subject entries
     -   For each `s` in `statement.subject`:
         -   For each digest (`alg`, `value`) in `s.digest`:
             -   If `alg` is in `acceptableDigestAlgorithms`:
                 -   If `hash(alg, artifactToVerify)` == `hexDecode(value)`:
-                    -   Add `s.name` to `artifactNames`
-    -   Reject if `artifactNames` is empty
+                    -   Add `s` to `matchedSubjects`
+                    -   Break
```

Comment:
remove; more than one entry can match

Reply:
This breaks on hash check, not entries.

Reply:
It's too confusing since there are multiple loops, and this is just pseudocode anyway. The logic is correct without the break, so I suggest omitting it.

In a future PR you might want to implement my suggestion to simplify the wording.

Reply:
I see what you mean. I'll update the wording now.

Reply:
Done, thanks!